From 850684fffa22b51fa037fda9780c1940e439df7f Mon Sep 17 00:00:00 2001
From: AlexTMjugador <AlexTMjugador@users.noreply.github.com>
Date: Wed, 15 Jul 2020 13:59:27 +0200
Subject: [PATCH] package/ramspeed: use a mirror for download

The original download URL was down for a bit of time when
building the package. Use a mirror instead.

Signed-off-by: AlexTMjugador <AlexTMjugador@users.noreply.github.com>
---
 package/ramspeed/ramspeed.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/ramspeed/ramspeed.mk b/package/ramspeed/ramspeed.mk
index 7dc4faae7c..5ab5ebf51e 100644
--- a/package/ramspeed/ramspeed.mk
+++ b/package/ramspeed/ramspeed.mk
@@ -5,7 +5,7 @@
 ################################################################################
 
 RAMSPEED_VERSION = 2.6.0
-RAMSPEED_SITE = http://www.alasir.com/software/ramspeed
+RAMSPEED_SITE = http://sources.buildroot.net/ramspeed
 RAMSPEED_ARCH = $(if $(BR2_i386),i386)$(if $(BR2_x86_64),x86_64)
 RAMSPEED_LICENSE = Alasir License
 RAMSPEED_LICENSE_FILES = LICENCE
-- 
2.27.0

